### PR TITLE
Update localization.rst

### PR DESCRIPTION
**Pull request by metanerd**
#### Summary
Howto for adding a new WIP language.

### CODE REVIEW

File: source/developer/localization.rst [hunk 1]
```diff
@@ -159,3 +159,9 @@ To grant trusted translators additional permissions as Weblate admin, add the us
 4. Go to **Groups**.
 5. Add the user to ``mattermost@TrustedReviewers`` group.
 6. Hit **Save**.
+
+To add a new language to the WIP project:
+
+1. Go to https://translate.mattermost.com/projects/i18n-wip/ .
```
Comment (justinegeffen) [inline source/developer/localization.rst:165]:
```suggestion
1. Go to `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`_.
```

File: source/developer/localization.rst [hunk 2]
```diff
@@ -159,3 +159,9 @@ To grant trusted translators additional permissions as Weblate admin, add the us
...
+2. Click on any of the three components **Tools > Start new translation** and add the languages you want to add and click **Save**.
```
Comment (justinegeffen) [inline source/developer/localization.rst:166]: Which three components are there? Or can the user just click "Tools"?

File: source/developer/localization.rst [hunk 3]
```diff
@@ -159,3 +159,9 @@ To grant trusted translators additional permissions as Weblate admin, add the us
...
+3. Repeat for the rest of the components or it could happen that Weblate already creates the language for the components in the same project.
```
Comment (justinegeffen) [inline source/developer/localization.rst:167]: If Weblate already creates the language does that mean the language was already there and not listed? Or is this a step to update the language in the other components?

File: source/developer/localization.rst (hunk 2 above)
Comment (metanerd) [reply source/developer/localization.rst:166]: No, the user has to click one component.

File: source/developer/localization.rst (hunk 3 above)
Comment (metanerd) [reply source/developer/localization.rst:167]: This is a step to create the language for other components. Sometimes it is there already, sometimes not. It is not made clear by weblate why.

File: source/developer/localization.rst [hunk 4]
```diff
@@ -159,3 +159,9 @@ To grant trusted translators additional permissions as Weblate admin, add the us
 4. Go to **Groups**.
 5. Add the user to ``mattermost@TrustedReviewers`` group.
 6. Hit **Save**.
+
+To add a new language to the WIP project:
```
Comment (jasonblais) [inline source/developer/localization.rst:163]:
```suggestion
To add a new language:
```

File: source/developer/localization.rst (hunk 2 above)
Comment (jasonblais) [inline source/developer/localization.rst:166]:
```suggestion
2. Click any of the components on the resulting page, then go to **Tools > Start new translation**. Select the languages you want to add, then click **Start new translation**.
```

File: source/developer/localization.rst (hunk 3 above)
Comment (jasonblais) [inline source/developer/localization.rst:167]:
```suggestion
3. Repeat step 2 for every component in `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`
```

**Review by jasonblais — CHANGES_REQUESTED** [review]
Thank you Elisabeth!

**Review by jasonblais — APPROVED** [review]

File: source/developer/localization.rst [hunk 5]
```diff
@@ -159,3 +159,9 @@ To grant trusted translators additional permissions as Weblate admin, add the us
 4. Go to **Groups**.
 5. Add the user to ``mattermost@TrustedReviewers`` group.
 6. Hit **Save**.
+
+To add a new language:
+
+1. Go to `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`_.
+2. Click any of the components on the resulting page, then go to **Tools > Start new translation**. Select the languages you want to add, then click **Start new translation**.
+3. Repeat step 2 for every component in `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`
```
Comment (justinegeffen) [inline source/developer/localization.rst:167]:
```suggestion
3. Repeat step 2 for every component in `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`_.
```

File: source/developer/localization.rst [hunk 6]
```diff
@@ -159,3 +159,9 @@ To grant trusted translators additional permissions as Weblate admin, add the us
 4. Go to **Groups**.
 5. Add the user to ``mattermost@TrustedReviewers`` group.
 6. Hit **Save**.
+
+To add a new language:
+
+1. Go to `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip/>`_.
```
Comment (justinegeffen) [inline source/developer/localization.rst:165]:
```suggestion
1. Go to `https://translate.mattermost.com/projects/i18n-wip/<https://translate.mattermost.com/projects/i18n-wip>`__.
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks, @metanerd, this is super-helpful. :)

**Pull request merged**